zproject/urls.py

```diff
@@ -78,7 +79,7 @@
         zerver.views.zephyr.webathena_kerberos_login,
         name='zerver.views.zephyr.webathena_kerberos_login'),
 
-    url(r'^accounts/password/reset/$', password_reset,
+    url(r'^accounts/password/reset/$', rate_limit()(password_reset),
```

Comment:
See above re a different potential spam issue -- this won't be an effective way to solve that problem, so let's leave it aside for this PR.